ios/Classes/PTFlutterViewController.h

```diff
@@ -14,6 +14,21 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) BOOL needsRemoteDocumentLoaded;
 @property (nonatomic) BOOL documentLoaded;
 
+// viewer options
+@property (nonatomic, assign, getter=isReadOnly, setter=setReadOnly:) BOOL readOnly;
+@property (nonatomic, assign, getter=isThumbnailViewEditingEnabled, setter=setThumbnailViewEditingEnabled:) BOOL thumbnailViewEditingEnabled;
+@property (nonatomic, copy, getter=getAnnotationAuthor) NSString * annotationAuthor;
+@property (nonatomic, assign, getter=isContinuousAnnotationEditing, setter=setContinuousAnnotationEditing:) BOOL continuousAnnotationEditing;
+
+- (void)setAnnotationAuthor:(NSString*)annotationAuthor;
+
+- (void)initViewerSettings;
+- (void)applyViewerSettings;
+
+@end
+
+@interface FLPTThumbnailsViewController : PTThumbnailsViewController
```

Comment:
You don't really need to prefix the prefix, just `FLThumbnailsViewController` would be fine and easier to read.